ble_controller/CHANGELOG.rst

```diff
@@ -5,6 +5,19 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+ble_controller 0.3.0-3.prealpha
+*******************************
+
+Changes
+=======
+* nRF52833 is now supported.
+
+Bugfixes
+========
+
+* Fixed an issue where ``hci_data_get()`` could return "No data available" when there was data available.
```

Comment:
Should be `hci_data_packet_get`.

Reply:
No, it is `hci_data_get()`, see https://github.com/NordicPlayground/nrfxlib/blob/master/ble_controller/include/ble_controller_hci.h#L123

Reply:
Ok. was looking at the wrong header. Thank you.